platform_versioned_docs/version-24.1/seqerakit/commands.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Commands"
+description: "seqerakit command options"
+date: "21 Oct 2024"
+tags: [seqerakit, cli, automation, commands]
+---
+
+Use the `--help` or `-h` parameter to list available commands and options:
+
+```bash
+seqerakit --help
+```
+
+### Input
+
+Seqerakit supports input through paths to YAML configuration files or directly from standard input (stdin).
+
+- Using file path:
+
+    ```bash
+    seqerakit /path/to/file.yaml
+    ```
+
+- Using stdin:
+
+    ```console
+    $ cat file.yaml | seqerakit -
+    ```
+
+See [Define your YAML file using CLI options](#define-your-yaml-file-using-cli-options) for guidance on formatting your input YAML files.
+
+### Dryrun
+
+Confirm that your configuration and command are correct before creating resources in your Seqera account, particularly when automating the end-to-end creation of multiple entities at once. To print the commands that would be executed with Platform CLI when using a YAML file, run your `seqerakit` command with the `--dryrun` flag:
+
+```bash
+seqerakit file.yaml --dryrun
+```
+
+### Specify targets
+
+When using a YAML file as input that defines multiple resources, use the `--targets` flag to specify which resources to create. This flag accepts a comma-separated list of resource names.
+
+For example, given a `test.yml` file that defines the following resources:
+
+```yaml
+workspaces:
+  - name: 'workspace-1'
+    organization: 'seqerakit'
+...
+compute-envs:
+  - name: 'compute-env'
+    type: 'aws-batch forge'
+    workspace: 'seqerakit/workspace-1'
+...
+pipelines:
+  - name: "hello-world"
+    url: "https://github.com/nextflow-io/hello"
+    workspace: 'seqerakit/workspace-1'
+    compute-env: "compute-env"
+...
+```
+
+You can target the creation of `pipelines` only by running:
+
+```bash
+seqerakit test.yml --targets pipelines
+```
+
+This will process only the pipelines block from the YAML file and ignore `workspaces` and `compute-envs`.
+
+To create both workspaces and pipelines, run:
+
+```bash
+seqerakit test.yml --targets workspaces,pipelines
```

Comment:
Do we explicitly say to list resources without spaces like this? (workspaces,pipelines,xxxx)

Reply:
Yes, we refer to this as a comma-separated list elsewhere.